scripts/stage

```diff
@@ -48,6 +48,9 @@ usage() {
 stage_s3() {
 	tarball="$(mktemp)"
 	tarball_md5="$(mktemp)"
+    tarball_manifest="$(mktemp)"
```

Comment:
This should be removed along with the others in lines 66-67.

Frankly this whole function should look more like `stage_windows_s3` and include a `trap` for the cleanup steps.